tests/queries/0_stateless/00965_shard_unresolvable_addresses.sql

```diff
@@ -1,4 +1,4 @@
--- Tags: shard
+-- Tags: shard, no-parallel
```

Comment:
Please, when you add tags like `no-parallel`, leave a comment explaining why. Some of the changes make no sense to me and have all test runs slower

Reply:
This test does "SYSTEM DROP DNS CACHE" which is shared ddl. Others are high-load tests